enclave/crypto/openssl/symcrypt_engine.c

```diff
@@ -2,14 +2,15 @@
 // Licensed under the MIT License.
 
 #include <openenclave/bits/defs.h>
+#include <openenclave/internal/crypto/init.h>
 
 /* When the enclave does not opt into the SymCrypt engine at link time,
  * the following implementation (weak) will be picked by the linker.
  * Note that we have to put the function in a separated source file from
  * init.c to prevent the linker pulls in the symbol along with the
  * oe_crypto_initialize function. */
-int _SYMCRYPT_ENGINE_Initialize()
+int _SC_OSSL_ENGINE_Initialize()
```

Comment:
```suggestion
int oe_SC_OSSL_ENGINE_Initialize()
```

The naming convention is to prefix non static functions with `oe_` prefix.